source/localizable/tutorial/autocomplete-component.md

```diff
@@ -144,11 +140,11 @@ We also define a `search` action here that is passed in to the component,
 For these actions to work, we need to modify the Mirage `config.js` file 
 to look like this, so that it can respond to our queries.
 
-```app/mirage/config.js
+```mirage/config.js
 export default function() {
   this.get('/rentals', function(db, request) {
     let rentals = [{
-        type: 'rentals',
```

Comment:
I know it's either for ED, and json:api doesn't specifically say it has to be plural, but all their [examples have it that way](http://jsonapi.org/format/#document-resource-objects). It's just a matter of consistency.